pettingzoo/utils/average_total_reward.py

```diff
@@ -1,9 +1,15 @@
+from __future__ import annotations
+
 import random
 
 import numpy as np
 
+from pettingzoo.utils.env import AECEnv
+
 
-def average_total_reward(env, max_episodes=100, max_steps=10000000000):
+def average_total_reward(
+    env: AECEnv, max_episodes: int = 100, max_steps: int = 10000000000
```

Comment:
Is this only for AEC?

Reply:
I think so, because the function calls env,last() which is only present in AEC environments

Reply:
Maybe add a comment explaining that? Just so in the future we don’t think this is a bug

Reply:
Changed the docstring from:

`Calculates the average total reward over the episodes`

To

`Calculates the average total reward over the episodes for AEC Environments`